ras_party/controllers/account_controller.py

```diff
@@ -462,7 +463,7 @@ def add_new_survey_for_respondent(payload, tran, session):
         business = query_business_by_party_uuid(business_id, session)
         if not business:
             raise ClientError("Could not locate business when creating business association",
-                              business_id=business_id,
+                              business_id=str(business_id),
```

Comment:
`business_id` comes from the response from the case service, so should already be a string